packages/components/src/components/custom-select-dropdown/custom-select-dropdown.scss

```
@@ -39,6 +39,12 @@
 
 	.db-input {
 		min-inline-size: variables.$db-sizing-2xl;
+
+		/* We don't need messages for search input */
```

why is there a validation message anyhow? This field wouldn't have any validation conditions like `pattern` or `required` defined. Investigating why this happens instead of hiding the error messages seems to be better to me.

---

fixed